test/e2e/csv_e2e_test.go

```diff
@@ -98,7 +99,7 @@ var _ = Describe("ClusterServiceVersion", func() {
 			}
 		})
 
-		It("is synchronized with the original csv", func() {
```

Comment:
I do think we want a test to make sure the copied CSV status is level with the "minimal copied CSV status" we've defined in this PR.